src/renderer/store/modules/utils.js

```diff
@@ -262,7 +257,7 @@ const actions = {
       }
 
       if (parsedString !== replaceFilenameForbiddenChars(parsedString)) {
-        reject(new Error('Forbidden Characters')) // use message as translation key
+        reject(new Error(i18n.t('Settings.Player Settings.Screenshot.Error.Forbidden Characters'))) // use message as translation key
```

Comment:
This comment is outdated now and could be removed.
```suggestion
        reject(new Error(i18n.t('Settings.Player Settings.Screenshot.Error.Forbidden Characters')))
```